Utilities/CodeGeneration/EdFi.Ods.CodeGen/Providers/Impl/DomainModelDefinitionProvidersProvider.cs

```diff
@@ -57,33 +60,47 @@ public IEnumerable<IDomainModelDefinitionsProvider> DomainModelDefinitionProvide
 
         private Dictionary<string, IDomainModelDefinitionsProvider> CreateDomainModelDefinitionsByPath()
         {
-            DirectoryInfo[] directoriesToEvaluate;
-            
+            DirectoryInfo[] directoriesToEvaluate = new DirectoryInfo[3];
+
             var domainModelDefinitionsByPath =
                 new Dictionary<string, IDomainModelDefinitionsProvider>(StringComparer.InvariantCultureIgnoreCase);
 
-            string edfiOdsImplementationApplicationPath = _solutionPath;
+            string edFiOdsImplementationApplicationPath = _solutionPath;
 
             string edFiOdsApplicationPath = _solutionPath.Replace(
                 CodeRepositoryConventions.EdFiOdsImplementationFolderName,
                 CodeRepositoryConventions.EdFiOdsFolderName);
 
-            if (_includePluginsProvider.IncludePlugins() && Directory.Exists(_extensionsPath))
+            directoriesToEvaluate = GetProjectDirectoriesToEvaluate(edFiOdsImplementationApplicationPath)
+                .Concat(GetProjectDirectoriesToEvaluate(edFiOdsApplicationPath))
+                .ToArray();
+
+            var extensionLocationPaths = _extensionLocationPluginsProviderProvider.GetExtensionLocationPlugins();
+
+            if (extensionLocationPaths != null && extensionLocationPaths.Any())
```

Comment:
this statement is redundant with the for each statement

Reply:
removed it